src/pages/CertificatesViewAll.js

```diff
@@ -20,8 +28,56 @@ const headersMap = {
     'Carbon Embodiment',
     'Status',
   ],
-  emma: ['Date', 'Commitment'],
-  reginald: [],
+  emma: [
+    'Date',
+    'Electric energy use',
+    'Production Start',
+    'Production End',
+    'Status',
+  ],
+  reginald: [
+    'Date',
+    'H2 Batch size',
+    'H2 Certificate holder',
+    'Carbon Embodiment',
+    'Status',
+  ],
+}
+
+const aggregateData = (data, id) => {
+  const rowsMap = {
+    heidi: (certs) =>
+      certs.map((cert) => [
+        <NameCell
+          key={cert.id}
+          date={formatDate(cert.created_at)}
+          name={formatCertName(cert)}
+        />,
+        `${cert.hydrogen_quantity_wh / 1000000} MWh`,
+        `${cert.energy_consumed_wh / 1000000} MWh`,
```

Comment:
Would it be better to add `.toFixed(n)` ?

Reply:
See above

Reply:
see above too